test/compress/exports.js

```diff
@@ -301,3 +322,35 @@ single_use_default: {
         f();
     }
 }
+
+single_use_class: {
+    options = {
+        reduce_vars: true,
+        toplevel: true,
+        unused: true,
+    }
+    input: {
+        export class A {}
+        A.prototype.p = "PASS";
+    }
+    expect: {
+        export class A {}
+        A.prototype.p = "PASS";
+    }
+}
+
+single_use_class_default: {
+    options = {
+        reduce_vars: true,
+        toplevel: true,
+        unused: true,
+    }
+    input: {
+        export default class A {}
+        A.prototype.p = "PASS";
+    }
```

Comment:
@kzc this is https://github.com/mishoo/UglifyJS/pull/4691#issuecomment-786196449